lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/six-month-fraud-check.yaml

```diff
@@ -0,0 +1,141 @@
+# Entry points
+
+START:
+  events:
+    next:
+      targetState: CONFIRM_NAME_DOB
+
+CONFIRM_NAME_DOB:
+  response:
+    type: page
+    pageId: pyi-confirm-name-dob
+  events:
+    next:
+      targetState: CONFIRM_ADDRESS
+    end:
+      targetState: UPDATE_NAME_DOB
+
+CONFIRM_ADDRESS:
+  response:
+    type: page
+    pageId: pyi-confirm-address
+  events:
+    fraud:
+      targetState: FRAUD_CHECK_6MFC
+    next:
+      targetState: ADDRESS_AND_FRAUD_6MFC
+
+UPDATE_NAME_DOB:
+  response:
+    type: page
+    pageId: pyi-update-name-dob
+  events:
+    end:
+      targetState: CONFIRM_NAME_DOB
+
+FRAUD_CHECK_6MFC:
+  response:
+    type: cri
+    criId: fraud
+  parent: CRI_STATE
+  events:
+    next:
+      targetState: EVALUATE_GPG45_SCORES_6MFC
+    enhanced-verification:
+      targetJourney: FAILED
+      targetState: FAILED_6MFC
+    end:
+      targetJourney: FAILED
+      targetState: FAILED_6MFC
+
+ADDRESS_AND_FRAUD_6MFC:
+  nestedJourney: ADDRESS_AND_FRAUD
+  exitEvents:
+    next:
+      targetState: EVALUATE_GPG45_SCORES_6MFC
+    enhanced-verification:
+      targetJourney: FAILED
+      targetState: FAILED_6MFC
+
+EVALUATE_GPG45_SCORES_6MFC:
+  response:
+    type: process
+    lambda: evaluate-gpg45-scores
+  events:
+    met:
+      targetState: IPV_SUCCESS_PAGE_6MFC
+      checkFeatureFlag:
+        ticfCriBeta:
+          targetState: CRI_TICF_BEFORE_SUCCESS
+    unmet:
+      targetJourney: FAILED
+      targetState: FAILED_6MFC
+    vcs-not-correlated:
+      targetJourney: FAILED
+      targetState: FAILED_6MFC
+
+CRI_TICF_BEFORE_SUCCESS_6MFC:
+  response:
+    type: process
+    lambda: call-ticf-cri
+  parent: CRI_TICF_STATE_6MFC
+  events:
+    next:
+      targetState: IPV_SUCCESS_PAGE_6MFC
+
+CRI_TICF_STATE_6MFC:
+  events:
```

Comment:
Does this also need a `next` event?

Reply:
Hm the example I copied from new-p2-identity doesn't have one of these either

Reply:
You're right it doesn't if it's a parent